### PR TITLE
common-tasks-keplr-explorer_mainnet

### PR DESCRIPTION
**Pull request by mmckennacudo**
Have removed screenshots that specify testnet from these instructions

### CODE REVIEW

File: src/.vuepress/config.js [hunk 1]
```diff
@@ -63,9 +63,9 @@ module.exports = {
         children: ['/build/tbcplaceholder.md', '/build/hardware.md'],
       },
       {
-        title: 'Funding & Earning with CUDOS Tokens',
+        title: 'Common Tasks with Keplr Wallet and Cudos Explorer',
```
Comment (ethanill) [inline src/.vuepress/config.js:66]: Replace with 'Using CUDOS' ? I think this is too wordy currently

Comment (mmckennacudo) [reply src/.vuepress/config.js:66]: I was originally going to name it 'Working with your CUDOS Tokens', but then decided to include keplr and explorer.. how about that?

Comment (ethanill) [reply src/.vuepress/config.js:66]: Maybe somewhere in between like 'Using CUDOS Tokens'? as not everyone reading this will own the token yet
